ocaml/idl/ocaml_backend/gen_api.ml

```diff
@@ -76,24 +76,23 @@ let gen_record_type ~with_module highapi tys =
       let all_fields = DU.fields_of_obj (Dm_api.get_obj_by_name highapi ~objname:record) in
       let field fld = OU.ocaml_of_record_field (obj_name :: fld.DT.full_name) in
       let map_fields fn = String.concat "; " (List.map (fun field -> fn field) all_fields) in
-      let regular_def fld = sprintf "%s : %s" (field fld) (OU.alias_of_ty fld.DT.ty) in
-
-      let make_of_field fld =
-        sprintf "\"%s\",rpc_of_%s x.%s" (String.concat "_" fld.DT.full_name)
-          (OU.alias_of_ty fld.DT.ty) (OU.ocaml_of_record_field (obj_name :: fld.DT.full_name))
-      in
-
-      let make_to_field fld =
-        sprintf "%s = %s_of_rpc (List.assoc \"%s\" x)" (field fld) (OU.alias_of_ty fld.DT.ty)
-          (String.concat "_" fld.DT.full_name)
+      let rpc_name fld = (String.concat "_" fld.DT.full_name) in
+      let get_default fld =
+        let default_value =
+          match fld.DT.ty with
+          | DT.Set (DT.Ref _) -> Some (DT.VSet [])
+          | _ -> fld.DT.default_value
+        in
+        match default_value with
+          None -> ""
+        | Some default -> sprintf "[@default (%s)]" (Datamodel_values.to_ocaml_string ~v2:true default)
```

Comment:
Jon said this is okay

Reply:
:)

Reply:
Unfortunately it doesn't but we have a plan to fix ocaml-rpc and make it possible